docs/data/sdks/typescript-react-native/index.md

```diff
@@ -77,7 +79,7 @@ init(API_KEY, 'user@amplitude.com', {
 
 ### Configuration
 
---8<-- "includes/sdk-ts-browser/shared-configurations.md"
+@{{ sdk_configuration_ts() }}
```

Comment:
Fixme: The macro needs to be updated to include all previous config options. Notice this uses "includes/sdk-ts-browser/shared-configurations.md" and the new sdk_configuration_ts() only has values from "includes/sdk-ts/shared-ts-configuration.md"